homeassistant/components/philips_js/media_player.py

```diff
@@ -192,45 +175,44 @@ def is_volume_muted(self):
 
     def turn_on(self):
         """Turn on the device."""
-        if self._on_script:
-            self._on_script.run(context=self._context)
-            self._update_soon(DELAY_ACTION_ON)
+        _LOGGER.debug("Firing event to turn on device")
```

Comment:
Is this debug line still needed?

Reply:
Well maybe not. It's hard to understand what is going on if you only are looking at logs for this component.